<launch>
  <arg name="namespace"               default="dynamixel_workbench"/>
  <arg name="is_loop"               default="false"/>

  <param name="$(arg namespace)/trajectory_info" value="$(find dynamixel_workbench_operators)/config/multi_joint_motion.yaml"/>
  <node name ="service_caller" pkg="dynamixel_workbench_operators" type="service_caller"/>
  <node name="multi_joint_operator" pkg="dynamixel_workbench_operators" type="multi_joint_operator"
        required="true" output="screen" ns="$(arg namespace)">
    <param name="is_loop" value="$(arg is_loop)"/>
  </node>
</launch>
